core/hoc/CompartmentMapping.hoc

```diff
@@ -104,10 +104,12 @@ proc registerMapping () { local cellIndex, activegid, nSoma, nAxon, nDend, nApic
 
   // list of gids on current rank
   gidvec = cellDistributor.getGidListForProcessor()
+  pop_info = new PythonObject()
 
   for cellIndex=0, gidvec.size()-1 {
     activegid = gidvec.x[cellIndex]
     cellref = cellDistributor.getCell(activegid)
+    pop_info = cellDistributor.getPopulationInfo(activegid)
```

Comment:
Hopefully one day we pass the final cell_distributor, instead of the global one, so we don't need to get `pop_info`. But I agree that is another work and for the moment this is the best